src/dao/MoodDao.java

```diff
@@ -1,7 +1,11 @@
 package dao;
 
 import foodmood.Mood;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
 import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Provides methods for reading mood objects from the database.
```

Comment:
Replace jsm158 with your id since you actually implemented this class. :)

Reply:
I'm not sure why it's not compiling.  I am using netbeans and it does compile for me locally.